### ISSUE TITLE
How to deal with motion data with Keras?

### ISSUE THREAD

**Issue by xietx1995** [keras-team/keras#11127]
I have 480 samples of BVH motion data for three sign language sentences, every 160 belongs to one sentence.  I want to use an RNN to classify them. I've padded and shuffled the data. I've tried to use k-fold to train an LSTM followed by a softmax to classify them. But it didn't work, and the validation accuracy is always zero.

The shape of the input data is (480, 856, 348) which means 480 samples, 856 timesteps for each sample, and 348 features. 

Here is a simple model that I used:

```
def build_model():
    model = Sequential()
    model.add(layers.LSTM(num_features))
    model.add(layers.Softmax(3))
    model.compile(optimizer=RMSprop(), loss='sparse_categorical_crossentropy', metrics=['acc'])
    return model
```

This is the code for training:

```
k = 4
num_val_samples = len(train_input) // k
num_epochs = 100
all_scores = []

for i in range(k):
    print('processing fold #', i)
    val_input = train_input[i * num_val_samples: (i + 1) * num_val_samples]
    val_output = train_output[i * num_val_samples: (i + 1) * num_val_samples]
    
    partial_train_input = np.concatenate([train_input[:i * num_val_samples], train_input[(i + 1) * num_val_samples:]])
    partial_train_output = np.concatenate([train_output[:i * num_val_samples], train_output[(i + 1) * num_val_samples:]])
    
    model = build_model()
    history = model.fit(
        partial_train_input,
        partial_train_output,
        epochs=num_epochs, 
        batch_size=1, 
        validation_data=(val_input, val_output))
    all_scores.append(history.history['acc'], history.history['val_acc'], history.history['loss'], history.history['val_loss'])
```
